examples/channel-hello-world/TARGETS

```diff
@@ -0,0 +1,28 @@
+cpp_binary(
+  name = 'tcp-server',
+  headers = subdir_glob([('.', '**/*.h')]),
+  srcs = [
+    'HelloStreamRequestHandler.cpp',
+    'HelloStreamSubscription.cpp',
+    'StreamHelloWorld_Server.cpp',
```

Comment:
I saw them, just moved them along for the ride assuming we'd fix when they become needed. Yes, I should have renamed them and forgot to :-/